index.html

```diff
@@ -47,7 +72,27 @@
 
 	</div>
 
```

Comment:
@nrobinson2000 Using `<br>` for anything but blocks of text content isn't really best practice, but I solved it by adding some bottom padding to the body.